Factory/JaegerTracerFactory.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Auxmoney\OpentracingBundle\Factory;
+
+use Exception;
+use OpenTracing\NoopTracer;
+use OpenTracing\Tracer;
+use Psr\Log\LoggerInterface;
+
+final class JaegerTracerFactory implements TracerFactory
+{
+    private $jaegerConfigFactory;
+    private $logger;
+
+    public function __construct(
+        JaegerConfigFactory $jaegerConfigFactory,
+        LoggerInterface $logger
+    ) {
+        $this->jaegerConfigFactory = $jaegerConfigFactory;
+        $this->logger = $logger;
+    }
+
+    public function create(string $projectName, string $agentHost, string $agentPort): Tracer
+    {
+        $tracer = new NoopTracer();
+
+        $config = $this->jaegerConfigFactory->create();
+        if (!$config) {
```

Comment:
will never happen, Config will always be created

Reply:
True